introduction/index.rst

```diff
@@ -245,4 +245,4 @@ Cadence xt-xcc or Cadence xt-clang?
 
 Is support for other toolchains forthcoming?
   Going forward, we would like to support the LLVM C compiler. Patches are
-  welcome.
\ No newline at end of file
+  welcome.
```

Comment:
Here is some unintended changes, I guess

Reply:
Thanks @ktrzcinx for the review. The couple newlines changes are intended.

While github doesn't show it well, the last line in this file had no newline at the end and this commit adds it. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

The only other whitespace changes are breaking down paragraphs that were entirely on one line, this will make `git diff` usable in the future. I did this only for the paragraphs I was already fixing anyway.

Reply:
Correction: one of the long paragraphs I broke down has no other, real change, my bad. I don't think it's worth it but I can undo this one of you insist .

Reply:
I won't insist.